src/DynamoCoreWpf/Views/Core/DynamoView.xaml.cs

```diff
@@ -929,13 +931,34 @@ private void DynamoViewModelRequestsFunctionNamePrompt(object sender, FunctionNa
         /// <returns></returns>
         internal void ShowNewFunctionDialog(FunctionNamePromptEventArgs e)
         {
-            var categorized =
-                SearchCategoryUtil.CategorizeSearchEntries(
-                    dynamoViewModel.Model.SearchModel.SearchEntries,
-                    entry => entry.Categories);
 
-            var allCategories =
-                categorized.SubCategories.SelectMany(sub => sub.GetAllCategoryNames());
+            List<string> addOns = new List<string>();
+            foreach (var element in dynamoViewModel.Model.SearchModel.SearchEntries)
+            {
+                // Only include packages and custom nodes
+                if (element.ElementType.HasFlag(ElementTypes.Packaged) || element.ElementType.HasFlag(ElementTypes.CustomNode))
```

Comment:
find anything weird using the searchElements directly?

Reply:
@mjkkirschner seems to work as expected.  Similar logic is used in a few other places.